spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -335,11 +346,14 @@ def step(  # noqa
                 base_action = (
                     np.array(base_action) * self.slowdown_base
                 )  # / self.ctrl_hz
+                print("Slow down...")
```

Comment:
Can we use loggers for this instead of print?

Reply:
Replaced print with self.say method